libsolidity/ast/Types.cpp

```diff
@@ -406,44 +578,59 @@ bool IntegerConstantType::isImplicitlyConvertibleTo(Type const& _convertTo) cons
 			return true;
 		return false;
 	}
+	else if (_convertTo.category() == Category::FixedPoint)
+	{
+		if (fixedPointType() && fixedPointType()->isImplicitlyConvertibleTo(_convertTo))
+			return true;
+		return false;
+	}
 	else if (_convertTo.category() == Category::FixedBytes)
 	{
 		FixedBytesType const& fixedBytes = dynamic_cast<FixedBytesType const&>(_convertTo);
-		return fixedBytes.numBytes() * 8 >= integerType()->numBits();
+		if (m_value.denominator() == 1)
+			return fixedBytes.numBytes() * 8 >= integerType()->numBits();
+		else
+			return fixedBytes.numBytes() * 8 >= fixedPointType()->numBits();
```

Comment:
Yes there is :) we talked about this. We take the fixed point bigint, we convert it over and take it in its pure byte form and pad right.